docs/source/topics/events.rst

```diff
@@ -70,6 +70,33 @@ and configure ``every_hour`` to be invoked once an hour,
 and ``every_two_hours`` to be invoked once every two hours.
 
 
+.. _cwe-events:
+
+Cloud Watch Events
+==================
+
+You can configure a lambda function to subscribe to
+any `Cloud Watch Event <https://amzn.to/2SCgWA6>`.
+
+To subscribe to a cloud watch event in chalice, you use the
+``@app.on_cw_event()`` decorator.  Let's look at an example.
+
+
+.. code-block:: python
+
+    app = chalice.Chalice(app_name='foo')
+
+    @app.on_cw_event({"source": ["aws.codecommit"]})
+    def on_code_commit_changes(event):
+        print(event.to_dict())
+
+In this example, we have a single lambda function that we subscribe to all
+events from the AWS Code Commit service. The first parameter to the decorator
+is the event pattern that will be used to filter the events sent to the function.
```

Comment:
It may be worth adding a link to the documentation for event patterns: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CloudWatchEventsandEventPatterns.html as it is not too obvious as to all the different options available.